core/rend/TexCache.cpp

```diff
@@ -401,22 +401,22 @@ void BaseTextureCacheData::ComputeHash()
 		old_texture_hash = old_vqtexture_hash;
 		old_vqtexture_hash ^= tcw.full & tcwMask;
 		// New hash
-	    XXH32_state_t *state = XXH32_createState();
-	    XXH32_reset(state, 7);
+	    XXH3_state_t *state = XXH3_createState();
+	    XXH3_64bits_reset(state);
```

Comment:
All the changes in this file must be reverted to preserve existing texture hash values.